apps/fluent-tester/src/theme/ThemePickers.android.tsx

```diff
@@ -2,8 +2,6 @@ import * as React from 'react';
 import { View } from 'react-native';
 import { TextV1 as Text } from '@fluentui-react-native/text';
 import { lightnessOptions, testerTheme } from './CustomThemes';
-import { themeChoices, ThemeNames } from './applyTheme';
-import { brandOptions, OfficeBrand } from './applyBrand';
```

Comment:
This is a large change for a PR I expected to only update checkbox tokens. Is the expectation that Android will not support other themes / changing the brand ramp? Or just that these themes weren't relevant so you want to remove them?

Reply:
@Saadnajmi  , correct. 
On android we're not going to have brandramp but only mode as "dark" or "light" with default brandCommunication palette. 
@ankraj12  , is my understanding correct here. 
Maybe we can have another PR to tackle the android changes into tester app ?

Reply:
Reverted for now.